README.md

```diff
@@ -1 +1,8 @@
 # gluster-mixins
+## Prerequisites
+1. Jsonnet
```

Comment:
Can you put a link from where jsonnet can be installed?

Reply:
ack.